sonic_platform_base/sonic_xcvr/mem_maps/public/cmis.py

```diff
@@ -57,17 +58,14 @@ def __init__(self, codes):
             CodeRegField(consts.MODULE_MEDIA_INTERFACE_PASSIVE_COPPER, self.getaddr(0x0, 87), self.codes.PASSIVE_COPPER_MEDIA_INTERFACE),
             CodeRegField(consts.MODULE_MEDIA_INTERFACE_ACTIVE_CABLE, self.getaddr(0x0, 87), self.codes.ACTIVE_CABLE_MEDIA_INTERFACE),
             CodeRegField(consts.MODULE_MEDIA_INTERFACE_BASE_T, self.getaddr(0x0, 87), self.codes.BASE_T_MEDIA_INTERFACE),
-            NumberRegField(consts.MEDIA_LANE_COUNT, self.getaddr(0x0, 88), 
+            NumberRegField(consts.MEDIA_LANE_COUNT, self.getaddr(0x0, 88),
                 *(RegBitField("Bit%d" % (bit), bit) for bit in range (0, 4))
             ),
-            NumberRegField(consts.HOST_LANE_COUNT, self.getaddr(0x0, 88), 
+            NumberRegField(consts.HOST_LANE_COUNT, self.getaddr(0x0, 88),
                 *(RegBitField("Bit%d" % (bit), bit) for bit in range (4, 8))
             ),
             NumberRegField(consts.HOST_LANE_ASSIGNMENT_OPTION, self.getaddr(0x0, 89), format="B", size=1),
-            NumberRegField(consts.MEDIA_LANE_ASSIGNMENT_OPTION, self.getaddr(0x1, 176), format="B", size=1),
             CodeRegField(consts.MEDIA_INTERFACE_TECH, self.getaddr(0x0, 212), self.codes.MEDIA_INTERFACE_TECH),
-            NumberRegField(consts.HW_MAJOR_REV, self.getaddr(0x1, 130), size=1),
-            NumberRegField(consts.HW_MINOR_REV, self.getaddr(0x1, 131), size=1),
             NumberRegField(consts.CMIS_MAJOR_REVISION, self.getaddr(0x0, 1),
```

Comment:
did you verify whether CMIS versions are dumped for DAC?. It crashed earlier in passive cables.

Reply:
yes, its not crashing for DAC.